client/src/api/ImageRepository.ts

```diff
@@ -0,0 +1,56 @@
+import { ApiResponse, ID } from '../types/types.ts';
+import ImageApiPaths from './ImageApiPaths.ts';
+import { fetchData } from './fetchData.ts';
+
+interface IImageRepository {
+  getImageById: (id: ID) => Promise<ApiResponse>;
+  createImageForChat: (chatId: ID, imageData: any) => Promise<ApiResponse>;
+  createImageForUser: (userId: ID, imageData: any) => Promise<ApiResponse>;
+  updateImage: (id: ID, imageData: any) => Promise<ApiResponse>;
+}
+
+const ImageRepository: IImageRepository = {
+  getImageById: async (id) => {
+    return await fetchData(ImageApiPaths.GET.BY_ID(id), {
+      method: 'GET',
+    });
+  },
+
+  createImageForChat: async (chatId, imageData) => {
+    const options = {
+      method: 'POST',
+      body: JSON.stringify(imageData),
```

Comment:
z tego co wiem, base64 powinno normalnie przejść w body requestu, także git?
trzeba sprawdzic czy to dziala

jak nie to bedziemy kombinowac z multipartem